.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Please include a summary of the changes and the related issue (if any). Please also include relevant motivation and context.
+
+Fixes # (issue)
+
+# Type of change
+
+Please delete options that are not relevant.
```

Comment:
I don't think this comment is necessary since the options can just not be selected.

Reply:
👍